config/zones.json

```diff
@@ -1524,12 +1524,48 @@
     ]
   },
   "US-NY": {
+    "capacity": {
+      "coal": 1113,
+      "gas": 11686,
+      "hydro": 3538,
+      "hydro storage": 1297,
+      "nuclear": 5292,
+      "oil": 8260,
+      "solar": 250,
+      "wind": 1010,
+      "biomass": 50
+    },
     "parsers": {
       "production": "US_NY.fetch_production"
     },
     "flag_file_name": "us.png",
     "contributors": [
       "https://github.com/systemcatch"
+      "https://github.com/hanshyde"
+    ]
+  },
+  "US-PJM": {
+    "capacity": {
+      "biomass": null,
+      "coal": 65205,
+      "gas": 63903,
+      "geothermal": null,
+      "hydro": 8675,
+      "hydro storage": null,
+      "nuclear": 33164,
+      "oil": 9148,
+      "solar": null,
+      "wind": 1114,
+      "battery discharge": null,
+      "other": 9112,
+      "unknown": 700,
+    },
+    "flag_file_name": "us.png",
+    "parsers": {
```

Comment:
Should be removed as we don't have the parsers ready yet. This will crash the backend.

Reply:
copy, done - all lines related to "flag_file_name" & "parsers"